default/scripting/species/common/xenophobic.macros

```diff
@@ -53,7 +53,7 @@ XENOPHOBIC_SELF
             ]
             stackinggroup = "XENOPHOBIC_POP_SELF"
             accountinglabel = "XENOPHOBIC_LABEL_SELF"
-            priority = [[LATE_PRIORITY]]
+            priority = [[EARLY_TARGET_POPULATION_POST_SCALING_PRIORITY]]
```

Comment:
As effect depends on target population, should probably be given a later priority?

Reply:
in particular, as the effect depends on the running value of the target population, not the initial value of TargetPopulation, yes, it is best handled as a later priority as you did.